src/components/newsFeedV2/newsFeed.css

```diff
@@ -0,0 +1,54 @@
+@import '../app/variablesV2.css';
+
+.newsFeedBox {
+  padding: 0;
+  height: auto;
+  font-family: var(--content-font);
+}
+
+.emptyNews {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  font-family: var(--content-font);
+  font-size: var(--paragraph-font-size-l);
+  padding-top: 16px;
+}
+
+.liskLogo {
+  opacity: 0.2;
+}
+
+.container {
+  min-height: 578px;
+  overflow: hidden;
+  height: 578px;
+
+  & > div {
+    height: 100%;
+  }
+
+  & .newsWrapper {
+    box-sizing: border-box;
+    padding: 0 20px;
+  }
+}
+
+.newsFeed {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  height: 866px;
+}
+
+.showMoreAlign {
+  margin-top: auto;
+  align-self: flex-end;
+}
+
+.showMore {
```

Comment:
This class is not used

Reply:
and the showMore css class is used for the toggle part for now.
Ali want to keep the same behavior as before for the community feed and then he will decide if we keep it like this or if we use a component scroll

Reply:
You're right. I missed it.